samples/Solidity/RefundVault.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.11;
```

Comment:
It's a pretty rare case. It's only the second language we'd be adding that doesn't have an extension or a filename defined.